source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -49,6 +53,23 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
+        const version = pkg.version;
+        const versionSplit = version.split('.');
+
+        let patchSplit = [];
+        if (versionSplit[2]) {
+            patchSplit = versionSplit[2].split('-');
```

Comment:
instead of using `2`, could you use a constant variable with descriptive name, so we can update the value in one place and understands what this value means?